packages/wonder-blocks-tokens/src/tokens/sizing.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * The baseline for the size tokens.
+ */
+const baseline = 8;
+
+/**
+ * Converts a number (px) to a rem value.
+ */
+function convertUnitsToRem(value: number): string {
```

Comment:
Would this utility be helpful for consumers? Should we export it?

This reminds me of the idea @marcysutton shared around providing utilities for converting values when we were brainstorming around the sizing tokens!

Reply:
I don't see why not! In that case, I think it would be better to move this to the `util/` folder, wdyt?

Side-note: maybe the function should be renamed to `convertPxToRem()` as the input unit is always px.

Reply:
hmmm I take it back. I'd like to see first how people are using these tokens. I feel that we should encourage teams to try to only use these tokens directly, and if we expose these kind of utils, we would be opening the door to use any kind of arbitrary values.

Reply:
Fair enough! We can add the utilities as needed 😄